public/html/index.html

```diff
@@ -30,8 +30,10 @@
       crossorigin
       type="font/woff2"
     />
+    <script src="/assets/splash-screen.js"></script>
   </head>
-  <body>
+  <body class="splash-screen">
+    <script src="/assets/splash-screen.js"></script>
```

Comment:
Why do we add twice?  Also can't script go after body? Does it need to be blocking?

Reply:
I added twice to avoid white color flashing. otherwise it won't work

Reply:
Can you help me understand why it's needed to be imported twice? Why couldn't the code be ran from the same script twice?

Reply:
I just tested with this code and it also fixes the issue:
```html
<!doctype html>
<html class="mode__popup light splash-screen">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link href="/assets/base.css" rel="stylesheet" />
  </head>
  <body class="splash-screen">
    <div id="app"></div>
    <script src="/assets/splash-screen.js"></script>
    <script src="browser-polyfill.js"></script>
  </body>
</html>
```

Reply:
yeah, I check as well now and it works. I pretty much sure that while I worked on it, I needed to add 2 scripts. 
will remove. thanks Kyran